n3fit/src/n3fit/checks.py

```diff
@@ -448,3 +445,18 @@ def check_multireplica_qed(replicas, fiatlux):
     if fiatlux is not None:
         if len(replicas) > 1:
             raise CheckError("At the moment, running a multireplica QED fits is not allowed.")
+
+
+@make_argcheck
+def check_polarized_configs(fitting, fitbasis, positivity_bound):
+    if fitbasis.startswith("POLARIZED_"):
+        if positivity_bound is None:
+            raise CheckError(
+                "For polarized fits, the 'positivity_bound' key has to be defined in the runcard."
+            )
+        if positivity_bound.get("unpolarized_bc") is None:
+            raise CheckError(
+                "For polarized fits, the name of the PDF has to be defined in positivity_bound::unpolarized_bc."
+            )
+        if fitting.get("sum_rules") != "TSR":
```

Comment:
```suggestion
        if fitting.get("sum_rules", True) and fitting.get("sum_rules") != "TSR":
```

`sum_rules: False` might be useful for testing.

Reply:
Ah, that might be a solution indeed. Let me try.